drivers/place/bookings.cr

```diff
@@ -208,6 +211,11 @@ class Place::Bookings < PlaceOS::Driver
     schedule.in(1.seconds) { poll_events } unless (subscription = @subscription) && !subscription.expired?
   end
 
+  # Allow apps to search for attendees (to add to new bookings) via driver instead of via staff-api (as some role based accounts may not have MS Graph access)
+  def list_users(query : String? = nil, limit : UInt32? = 20_u32)
+    calendar.list_users(query, limit).get.as_a
```

Comment:
you don't actually need to call `.get.as_a` unless the function is used within the driver
this will be done automatically with more performance for any remote requests

```suggestion
    calendar.list_users(query, limit)
```

Reply:
Thanks! didn't know that